src/api/app/controllers/webui2/package_controller.rb

```diff
@@ -25,4 +25,24 @@ def webui2_statistics
                                                       repository: @repository,
                                                       architecture: params[:arch]).results
   end
+
+  def webui2_prefill
+    target_project = ''
+    target_package = ''
+    description = ''
+    cleanup_source = @project.branch? # We should remove the package if this request is a branch
+    linked_package = @package.backend_package.links_to
+    if linked_package
+      target_project = linked_package.project.name
+      target_package = linked_package.name
+      description = @package.commit_message(target_project, target_package)
```

Comment:
It looks like the description isn't filled. I just tried it again to be sure in the review app.

To reproduce this:
- Go to https://obs-reviewlab.opensuse.org/danidoni-submit-request-form-is-not-pre-filled-with-target-project/package/show/home:Admin:branches:home:Admin/ruby
- See the `I improved the text` in the `Latest Revision` box. (this should be the description)
- Click on `Submit package`
- The project and package fields are pre-filled. The description is not.

Reply:
This is based on `.changes` files, no on commit messages.

Reply:
We should rename the method from the `Package` model then. It's literally called `commit_message`.

Reply:
Feel free to refactor this, afterward :)

Reply:
https://github.com/openSUSE/open-build-service/issues/8306

Reply:
I'll check it out

Reply:
Don't bother with this now, we can tackle the issue later and it doesn't have to be in this deliverable.